lib/twingly/search/query.rb

```diff
@@ -2,25 +2,44 @@
 
 module Twingly
   module Search
+    # Twingly Search API query
+    #
+    # @attr [String] pattern the search query.
+    # @attr [String] language which language restrict the query to.
+    # @attr [Client] client the client that this query is connected to.
+    # @attr [#to_time] start_time search for posts published after this time.
+    # @attr [#to_time] end_time search for posts published after this time.
```

Comment:
are these inclusive or exclusive? I don't think it should say "to_time" on both either